packages/UI/src/components/atoms/Link/StyledLink.ts

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import theme from '../../../styles/theme';
```

Comment:
@alston-hsu 
importing the theme is highly discouraged, please use this approach https://styled-components.com/docs/advanced#function-themes